libs/core/src/lib/busy-indicator/busy-indicator.component.html

```diff
@@ -1,13 +1,12 @@
-<div
-    [ngClass]="{
+<div [ngClass]="{
         'fd-busy-indicator--l': size == 'l',
         'fd-busy-indicator--m': size == 'm',
         'fd-busy-indicator--s': size == 's'
     }"
-    *ngIf="loading"
-    [attr.aria-label]="loadingLabel"
->
+     *ngIf="loading"
+     aria-hidden="true"
```

Comment:
Should this be `aria-hidden="!loading"`? If this is hidden from screenreaders etc. all the time then the aria-label will always be ignored

Reply:
check whether this is fine in Firefox